src/Entity/Post.php

```diff
@@ -126,121 +127,103 @@ public function __construct()
         $this->tags = new ArrayCollection();
     }
 
-    public function getId()
+    public function getId(): int
     {
         return $this->id;
     }
 
-    public function getTitle()
+    public function getTitle(): ?string
```

Comment:
Not for this PR, but we should make the title (and slug) mandatory so this return value can never be null, don't you think?

Reply:
I thought the same at beginning, but they are already mandatory in DB & FormValidation. so when I tested the real app (`/es/admin/post/new`):

> Type error: Return value of App\Entity\Post::getTitle() must be of the type string, null returned

The reason is on form initialization, it needs `mapDataToForms()` and to read each defined form field from underlying object (default values), so `PropertyAccessor->readProperty(array(object(Post)), 'title')` calls to `Post->getTitle()` and boom! `null` value.

The solution could be set a fake or default title/slug for new `Post`s, now that `Slugger::slugify()` is static, it can be easier. WDYT? then we can change it to strict `string`.

Reply:
I'm not sure it's worth it. Now that you explained that, I remember that I faced this issue too a while ago. I needed to put `?string` in an entity because of the Form behavior that you mentioned. So let's keep it. Thanks.